README.md

```diff
@@ -14,7 +14,7 @@ The examples in this repository are structured as follows:
 - [Analog Hamiltonian Simulation](#ahs)
 - [Qiskit with Braket](#qiskit)
 - [CUDA-Q](#cudaq)
-
+- [Experimental Capabilities](#experimental)
```

Comment:
this anchor link doesn't work - can addressed in a follow-up
```suggestion
- [Experimental Capabilities](#experimental-dynamic)
```

Reply:
okay, acknowledged the issue thanks!